requirements.txt

```diff
@@ -3,5 +3,5 @@ Click<7
 coloredlogs
 marshmallow
 pyyaml
-SQLAlchemy
+SQLAlchemy<1.4
```

Comment:
Why is this necessary?

Reply:
I see, good catch 👍

Reply:
When we are done with the refactoring, we should be able to use the latest version